Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -635,10 +635,29 @@ public final class ManifestLoader: ManifestLoaderProtocol {
                         // This provides some safety against arbitrary code execution when parsing manifest files.
                         // We only allow the permissions which are absolutely necessary.
                         if self.isManifestSandboxEnabled {
-                            let cacheDirectories = [self.databaseCacheDir, moduleCachePath].compactMap{ $0 }
-                            let strictness: Sandbox.Strictness = toolsVersion < .v5_3 ? .manifest_pre_53 : .default
```

Comment:
This change isn't as material as it looks.  The pre-5.3 special case in the Sandbox was to allow the manifest to run interpreted, which is something that no longer happens regardless of the tools version.  This controlled writability to `org.llvm.clang` cache directories etc, and is not likely to affect existing manifests.

There is actually a question of whether sandbox rule changes should be based on tools version at all, as this makes it very easy to circumvent tightened sandboxes even in new builds of SwiftPM.

Reply:
I think we should probably double-check the assumption that this doesn't affect existing libraries by running through our various compatibility suites. I agree in principle that the rules shouldn't depend on the tools-version, on the other hand we need to make sure we don't break existing packages.

Reply:
That's a good point.

The other approach would be to add back the basic knobs-and-switches controlled by that specialized setting and just apply them at the call site.  I mainly wanted to get the "special sauce" out of the SandboxProfile type.